genre_prediction/.vscode/launch.json

```diff
@@ -22,5 +23,14 @@
             "console": "integratedTerminal",
             "justMyCode": false
         }
+        ,
+        {
+            "name": "streamlit_app",
+            "type": "python",
+            "request": "launch",
+            "program": "streamlit_app.py",
+            "console": "integratedTerminal",
+            "justMyCode": false
+        }
```

Comment:
if you feel that this is weird an look like the previous change, it is because it is the launch.json from the genre_prediction folder, not from the main folder.